downstream/assemblies/hub/assembly-syncing-to-cloud-repo.adoc

```diff
@@ -41,7 +41,23 @@ You can use underscores in the collection name if it improves readability.
 
 include::hub/con-remote-repos.adoc[leveloffset=+1]
 
-include::hub/proc-obtaining-org-collection-url.adoc[leveloffset=+1]
+// [hherbly: replacing this with the 3 modules below from the Getting started with hub guide include::hub/proc-obtaining-org-collection-url.adoc[leveloffset=+1]
+
+== Creating the API token in {HubName}
+
+Before you can interact with {HubName} by uploading or downloading collections, you must create an API token. The {HubName} API token authenticates your `ansible-galaxy` client to the Red Hat {HubName} server.
+
+Your method for creating the API token differs according to the type of {HubName} that you are using:
+
+* {HubNameStart} uses offline token management. See xref:proc-create-api-token[Creating the offline token in {HubName}].
+
+* {PrivateHubNameStart} uses API token management. See xref:proc-create-api-token-pah[Creating the API token in {PrivateHubName}].
+
+* If you are using Keycloak to authenticate your {PrivateHubName}, follow the procedure for xref:proc-create-api-token[Creating the offline token in {HubName}].
```

Comment:
I think it might be cleaner, especially from the point of view of converting to the new tooling, to put this in a separate module.